lib/travis/build/appliances/update_apt_keys.rb

```diff
@@ -22,8 +22,27 @@ def apply
               bash &>/dev/null
             KEYUPDATE
             sh.cmd command, echo: false
+
+            sh.cmd rabbit_key_update, echo: false
           end
         end
+
+        def rabbit_key_update
+          <<~EORABBIT
+          if [[ ( \"$TRAVIS_DIST\" == trusty || \"$TRAVIS_DIST\" == precise) && -f /etc/init.d/rabbitmq-server ]]; then
+            curl -sSL #{key_url('rabbitmq-trusty')} | sudo apt-key add -
```

Comment:
Is `rabbitmq-trusty` added via another PR e.g. keychain?

Reply:
Yes, https://github.com/travis-ci/apt-source-safelist/pull/397 merged the updated keys.